### DISCUSSION TITLE
What is the later version? 1.6.9 pre-release or the 1.6.39 of current release?

### DISCUSSION THREAD

**Discussion by sakuramaezonoYCU** [quarto-dev/quarto-cli#11555] [Q&A] [answered]
### Description

What is the later version? 1.6.9 pre-release or the 1.6.39 of current release?

**Comment by cderv** [comment, marked as answer]
Hi, 

Are you asking this question in a specific context ? 

You can find the latest version available at : https://quarto.org/docs/download/
It is marked current release. 

We also use github releases so this will give you the latest version: https://github.com/quarto-dev/quarto-cli/releases/latest

So I can confirm that 1.6.39 is the current release, out this week. 

1.7 series is now the quarto pre-release.

**Reply by sakuraerimaezono** [reply]
Thank you! I have been using the pre-release version 1.6.9 and wanted to update to the current but just wanted to make sure it is an actual update and not a downgrade. Just got a little bit confused with the version number.

**Reply by mcanouil** [reply]
Increasing numbers means upgrade.
